doc/sphinx/addendum/program.rst

```diff
@@ -194,14 +194,14 @@ Program Fixpoint
    The optional order annotation follows the grammar:
 
    .. productionlist:: orderannot
-      order      : measure `term` (`term`)? | wf `term` `term`
+      order      : measure `term` (`term`)? | wf `term` `ident`
```

Comment:
How do you parse this grammar? I personally understand these parentheses as part of the syntax, and the `?` too. The usual way of marking optional parts as documented in https://coq.inria.fr/refman/language/gallina-specification-language.html#about-the-grammars-in-the-manual is to use square brackets.

Reply:
Fixed.